Modules/Invoice/Resources/views/subviews/create/invoice-details.blade.php

```diff
@@ -108,13 +108,15 @@ class="ml-2"
                 <div class="form-group">
                     <label for="term_start_date" class="field-required">Term Start Date</label>
                     <input type="date" class="form-control" name="term_start_date" id="term_start_date" required="required"
-                        value="{{ now()->format('Y-m-d') }}">
+                           value="{{ \Carbon\Carbon::now()->subMonth()->startOfMonth()->toDateString() }}">
                 </div>
                 <div class="form-group">
                     <label for="term_end_date" class="field-required">Term End Date</label>
                     <input type="date" class="form-control" name="term_end_date" id="term_end_date" required="required"
-                        value="{{ now()->format('Y-m-d') }}">
+                           value="{{ \Carbon\Carbon::now()->subMonth()->endOfMonth()->toDateString() }}">
```

Comment:
now() is be default treated as a carbon function 
```suggestion
                           value="{{ now()->subMonthNoOverflow()->endOfMonth()->toDateString() }}">
```